app/konnect/org-management/teams-roles-users.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Manage Teams, Roles, and Users
+no_version: true
+---
+
+Many organizations have strict security requirements. For example, organizations
+need the ability to segregate the duties of an administrator to ensure that a
+mistake or malicious act by one administrator doesn’t cause an outage.
+
+To help secure and govern your environment, {{site.konnect_short_name}} provides
+the ability to manage authorization with teams and roles. You can use Konnect's
+predefined teams for a standard set of roles, or create custom teams with
+any roles you choose. Invite users and add them to these teams to manage user
+access.
+
+{:.note}
+> **Note:** If Okta integration is [enabled](/konnect/org-management/okta-idp),
+{{site.konnect_short_name}} users and teams become read-only. An organization
+admin can view all registered users in {{site.konnect_short_name}}, but cannot
+edit their team membership from the {{site.konnect_short_name}} side. To manage
+automatically-created users, adjust user permissions through Okta, or
+[adjust team mapping](/konnect/org-management/okta-idp/#map-teams-to-groups).
+
+## Prerequisites
+
+You must be part of the **Organization Admin** team to manage users, teams, and
+roles.
+
+## Manage teams and roles
+
+You can find a list of all teams in your organization through
+![](/assets/images/icons/konnect/icn-organizations.svg){:.inline .no-image-expand .konnect-icon}
+**Organization** > **Teams** in Konnect.
+
+* **Team:** A group of users with access to the same roles. Teams can be
+used to functionally separate departments or create hierarchical
```

Comment:
This implies that teams can contain teams to me. Is that the case?

Reply:
This is not yet the case. I'll rephrase.